src/pages/what-we-are-doing.js

```diff
@@ -1,117 +1,176 @@
+
+    <VerticalSpacing size={10} />
+
+    <div className="carousel">
+      <Malarquee fill hoverToPause rate={25}>
+        <div className="logos">
+          {collaborationsMap.map(({ id, ...item }) => (
+            <img
+              key={id}
+              className={classNames('logo', id)}
+              {...item}
+              alt={item.alt || ''}
+            />
+          ))}
+        </div>
+      </Malarquee>
+    </div>
+
+    <VerticalSpacing size={10} />
+
+    <ConstrainedWidth>
+      <Text size="large">
+        <H withArcs>Recent Projects</H>
+      </Text>
     </ConstrainedWidth>
 
-    <SquiggleSeparator></SquiggleSeparator>
-
-    <HexagonGrid rate={25} rows={2}>
-      <Hexagon size={80} grey src={castLogo} />
-      <Hexagon size={80} grey src={comicReliefLogo} />
-      <Hexagon size={80} grey src={mhfLogo} />
-      <Hexagon size={80} grey src={phfLogo} />
-    </HexagonGrid>
-
-    <SquiggleSeparator />
-
-    <StaticQuery
-      query={graphql`
-        query {
-          projects: allMarkdownRemark(
-            filter: { fields: { sourceName: { eq: "projects" } } }
-          ) {
-            nodes {
-              frontmatter {
-                title
-                problem
-                solution
-                subtitle
-                image {
-                  childImageSharp {
-                    fluid(maxWidth: 768) {
-                      ...GatsbyImageSharpFluid
+    <VerticalSpacing size={4} />
+
+    <Section>
+      <StaticQuery
+        query={graphql`
+          query {
+            projects: allMarkdownRemark(
+              filter: { fields: { sourceName: { eq: "projects" } } }
+            ) {
+              nodes {
+                frontmatter {
+                  title
+                  problem
+                  solution
+                  subtitle
+                  image {
+                    childImageSharp {
+                      fluid(maxWidth: 768) {
+                        ...GatsbyImageSharpFluid
+                      }
                     }
                   }
                 }
               }
             }
           }
-        }
-      `}
-      render={data => {
-        return data.projects.nodes.map((project, index) => (
-          <WorkDescription
-            alternate={index % 2 === 0}
-            key={project.frontmatter.title}
-            id={slugify(project.frontmatter.title, '-')}
-            fluid={project.frontmatter.image.childImageSharp.fluid}
-            level={3}
-            title={project.frontmatter.title}
-            subtitle={project.frontmatter.subtitle}
-            problem={project.frontmatter.problem}
-            solution={project.frontmatter.solution}
-          />
-        ))
-      }}
-    />
-    <SquiggleSeparator />
+        `}
+        render={data => {
+          return data.projects.nodes.map((project, index) => (
+            <React.Fragment key={project.frontmatter.title}>
+              {index !== 0 ? (
```

Comment:
I'd love a .join for arrays that could do it using react instead of strings :thinking:

Reply:
haha I thought the same. It looks like Internet people have achieved this with `reduce` and writing a helper fn. Reminds me a little of how `map` is abused to async filter :smile: